platsec.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+ARTIFACTS_DIR="grype-vuln-artifacts"
+IMAGE_TAG="quay.io/gburges/kibana:latest"
+
+mkdir -p "$ARTIFACTS_DIR"
+
+#install and run syft
+curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b ./bin
```

Comment:
I think you can wrap these in a "download binary" function - that takes the URL and the binary as parameters - so that you can do a small logic block that checks if either of the commands are present in the machine, and if not, they are downloaded. 

You can store the commands in variables so that when you actually use them you invoke the variable directly,

Reply:
i thought the workspaces get wiped after every run, so it won't make sense to check for existence imo. I did wrap the download into a function, update inc

Reply:
I understand, and you're right, the workspaces are wiped on every run, on the CI pipeline, but if I want to run this script myself, and I happen to have the command available (not even locally, perhaps I have it installed somewhere and it's available in my `PATH`) then I don't need to download it every time. This is what I was thinking about when I made the comment. 

I would expect to be able to run this locally and in the CI , and I think it is of great value to be able to handle both situations. 

I would also check if the `bin` directory exists - or move it under any other directory too. The good thing is that it doesn't matter where you put the binaries, as long as you're able to reference them with their path.